EntityRecognitionCustomSkill/function_app.py

```diff
@@ -0,0 +1,136 @@
+import os
+import azure.functions as func
+import logging
+import json
+import jsonschema
+from azure.ai.inference import ChatCompletionsClient
+from azure.core.credentials import AzureKeyCredential
+
+app = func.FunctionApp(http_auth_level=func.AuthLevel.ANONYMOUS)
+
+# A healthcheck endpoint. Important to make sure that deployments are healthy.
+# It can be accessed via <base_url>/api/health
+@app.route(route="health", auth_level=func.AuthLevel.ANONYMOUS)
+def HealthCheck(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info("Calling the healthcheck endpoint")
+    response_body = {"status": "Healthy"}
+    response = func.HttpResponse(
+        json.dumps(response_body, default=lambda obj: obj.__dict__)
+    )
+    response.headers["Content-Type"] = "application/json"
+    return response
+
+
+@app.function_name(name="EntityRecognition")

```

Comment:
Are you thinking of there being multiple such functions in this same class?

I just ask because the directory structure doesn't make that obvious

Reply:
yes, so based on your feedback it makes sense for me to have 3 separate functions for each of the models that the Azure function will call.